GOVERNANCE.md

```diff
@@ -1,45 +1,106 @@
 
-## Community Structure
+The project asks its users to participate in the project and community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user contributions include (but are not limited to):
 
-![community organization](/media/governance/community_organization.png)
+- evangelising about the project (e.g. a link on a website and word-of-mouth awareness raising)
+- informing developers of strengths and weaknesses from a new user perspective
+- providing moral support
+- providing financial support
 
-The community structure comprises two parts. The first part is the community organization, and the second part is the role division within the community group and the promotion path of each community role.
+Users who continue to engage with the project and its community will often become more and more involved. Such users may find themselves becoming contributors, as described in the above section.
 
-PMC: The Project Management Committee; functions as the core management team that oversees the TiDB community. The PMC decides the roadmap of the project and makes major decisions related to the community.
+## Approving PRs
 
-Developer Group:
+PRs may be merged only after receiving at least two approvals (LGTMs) from Reviewers, more previliged roles from the relevant SIGs, or even Maintainers.
 
-- Maintainer: Mainatiners are the planners and designers of the project, with the authority to merge branches into the master. 
-- Committer:  Recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A Committer must have completed at least one feature or fix a major bug independently.
-- Reviewer: Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors. Reviewer has the rights to approve code reviews with LGTM (Looks Good To Me).
-- Active contributor: Contributed 8 PRs and above to TiKV within one year and received recognition from 2 incumbent Reviewers
-- Contributor: Merged at least 1 PR in any Repo under [TiKV](https://github.com/tikv) 
+## Decision making and voting
 
-Organizer Committee:  The Organizer Committee consists of organizers in charge of event or activity operations; in charge of the execution of PMC’s strategies and decisions. Specifically, the members are the leaders of user groups or community event organizers across regions
+Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. Any of the day-to-day project maintenance can be done by a [lazy consensus](http://communitymgt.wikia.com/wiki/Lazy_consensus) model. Cross-SIG or community-level decisions must be brought to broader awareness via community meetings, slack channels, etc.
```

Comment:
Is an RFC enough? I think it should be. There should probably be an async way to communicate this sort of thing, e.g. via the monthly newsletter (perhaps that is too low frequency?)

Reply:
Yeah  monthly community meeting and our monthly newsletter, plus slack should be enough for async communication

Reply:
Community meeting and Slack are both synchronous communication (because somebody in a different timezone might miss things). Monthly newsletter should be enough for async, we can experiment.